torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3059,181 +3108,148 @@ def _make_tensor_helper(shape, low=None, high=None):
+           dtypesIfCPU=all_types_and(torch.bool, torch.half, torch.bfloat16),
+           dtypesIfCUDA=all_types_and(torch.bool, torch.half, torch.bfloat16),
+           test_inplace_grad=True,
+           safe_casts_outputs=True,
+           sample_inputs_func=sample_inputs_xlogy),
+    OpInfo('trace',
+           dtypes=all_types_and_complex(),
+           dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half),
+           test_inplace_grad=False,
+           supports_out=False,
+           sample_inputs_func=sample_inputs_trace),
+    UnaryUfuncInfo('sigmoid',
+                   ref=scipy_reference_wrapper(reference_sigmoid),
+                   decorators=(precisionOverride({torch.float16: 1e-2,
+                                                  torch.bfloat16: 1e-2}),),
+                   skips=(
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_extremal',
+                                device_type='cpu', dtypes=[torch.cfloat, torch.cdouble]),
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_hard',
+                                device_type='cpu', dtypes=[torch.cfloat, torch.cdouble]),
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_normal',
+                                device_type='cpu', dtypes=[torch.cfloat, torch.cdouble]),
+                       # RuntimeError: sigmoid does not support automatic differentiation for outputs with complex dtype.
+                       SkipInfo('TestCommon', 'test_variant_consistency_jit',
+                                dtypes=[torch.complex64, torch.complex128]),
+                       SkipInfo('TestCommon', 'test_variant_consistency_eager',
+                                dtypes=[torch.complex64, torch.complex128]),),
+                   dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16),
+                   dtypesIfCPU=all_types_and_complex_and(torch.bool, torch.bfloat16),
+                   dtypesIfCUDA=all_types_and(torch.bool, torch.half, torch.bfloat16),
+                   safe_casts_outputs=True,
+                   assert_autodiffed=True,
+                   test_complex_grad=False),  # Reference: https://github.com/pytorch/pytorch/issues/48552
+    UnaryUfuncInfo('digamma',
+                   ref=scipy_reference_wrapper(lambda x: scipy.special.digamma(x)),
+                   decorators=(precisionOverride({torch.float16: 5e-1}),),
+                   dtypes=all_types_and(torch.bool),
+                   dtypesIfCPU=all_types_and(torch.bool),
+                   dtypesIfCUDA=all_types_and(torch.bool, torch.half),
+                   skips=(
+                       # In some cases, output is NaN (for input close to
+                       # negative integers) especially due to reduced precision
+                       # in float16 and NaN's can't be tested for equality.
+                       SkipInfo('TestCommon', 'test_variant_consistency_jit',
```

Comment:
This skip is not needed but a new skip may be

Reply:
Have pulled the latest master and just update the `ref` arguments with wrapped. Kept the other stuff as is in the master.